src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-cluster/nfs-cluster.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, NgZone, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { NfsService } from '~/app/shared/api/nfs.service';
+import { ListWithDetails } from '~/app/shared/classes/list-with-details.class';
+import { ActionLabelsI18n } from '~/app/shared/constants/app.constants';
+import { CdTableAction } from '~/app/shared/models/cd-table-action';
+import { CdTableColumn } from '~/app/shared/models/cd-table-column';
+import { CdTableSelection } from '~/app/shared/models/cd-table-selection';
+import { Permission } from '~/app/shared/models/permissions';
+import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
+import { URLBuilderService } from '~/app/shared/services/url-builder.service';
+import { NFSClusterListConfig, NFSClusterTableModel } from '../models/nfs-cluster-config';
+import { OrchestratorStatus } from '~/app/shared/models/orchestrator.interface';
+import { OrchestratorService } from '~/app/shared/api/orchestrator.service';
+const BASE_URL = 'cephfs/nfs';
+@Component({
+  selector: 'cd-nfs-cluster',
+  templateUrl: './nfs-cluster.component.html',
+  styleUrls: ['./nfs-cluster.component.scss'],
+  providers: [{ provide: URLBuilderService, useValue: new URLBuilderService(BASE_URL) }]
+})
+export class NfsClusterComponent extends ListWithDetails implements OnInit {
+  @ViewChild('test', { static: true })
+  test: TemplateRef<any>;
+
+  @ViewChild('ipPorts', { static: true })
+  ipPorts: TemplateRef<any>;
```

Comment:
don't forget to user clear name for each items

```suggestion
  @ViewChild('hostnameTpl', { static: true })
  hostnameTpl: TemplateRef<any>;

  @ViewChild('ipAddrTpl', { static: true })
  ipAddrTpl: TemplateRef<any>;
```

Reply:
updated